app/src/main/java/com/azuresamples/msalnativeauthandroidkotlinsampleapp/AccessApiFragment.kt

```diff
@@ -22,6 +24,7 @@ import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.launch
 import kotlinx.coroutines.withContext
+import okhttp3.Response
 
 class AccessApiFragment : Fragment() {
```

Comment:
Let's add some Javadoc explaining what this fragment does (what use cases it demos), especially as we have no docs for this.

Reply:
cc @nilo-ms @borgesmb you may want to add this on iOS as well.